app/packages/operators/src/OperatorInvocationRequestExecutor.tsx

```diff
@@ -33,7 +33,12 @@ function RequestExecutor({ queueItem, onSuccess, onError }) {
       callback: queueItem.callback,
       ...(queueItem?.request?.options || {}),
     });
-  }, []);
+  }, [
+    executor,
```

Comment:
This will likely cause infinite execution of this operator

Reply:
Ya I thought this was a weird linting error. The intent is `on load run this` but its insisting on either removing the `[]` or adding all the deps...?

Reply:
<img width="1329" alt="image" src="https://github.com/voxel51/fiftyone/assets/462228/23e16493-8c4c-4cd2-81db-677a9a9e0242">